From b80f46f17d17b0a80f03d61128dff9475059c621 Mon Sep 17 00:00:00 2001
From: Evan Wang <xswang@marvell.com>
Date: Fri, 16 Jun 2017 14:27:32 +0800
Subject: [PATCH 1146/1345] dts: cp110: comphy: add property of dfx-reg and
 sar-data

commit  4d06023adc41412672f3a00a049cb6e6cfbe4c64 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

DFX register is needed to configure PCIe clock MUX
when clock direction is out and lane5 is indicated as
clock source. The property of dfx-reg indicates the DFX
register space information.

Sample At Reset(SAR) is needed to get PCIe clock direction
when COMPHY is initialized for PCIe. The property of
sar-data is the phandle to SAR device in DTS.

Change-Id: I4eb27dd7330c53b62a9d4d78f9866ffd9f230445
Signed-off-by: Evan Wang <xswang@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/40539
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi |    2 ++
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi   |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index a652861..cb3309c 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -375,7 +375,9 @@ comphy1: comphy {
 	compatible = "marvell,cp110-comphy";
 	reg = <0x441000 0x100>, <0x120000 0x6000>;
 	reg-names = "comphy", "serdes";
+	dfx-reg = <0x400280 4>;
 	#phy-cells = <2>;
+	sar-data = <&cps_sar>;
 	status = "disabled";
 };
 
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 69c35c4..1d415d8 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -473,7 +473,9 @@ comphy0: comphy {
 	compatible = "marvell,cp110-comphy";
 	reg = <0x441000 0x100>, <0x120000 0x6000>;
 	reg-names = "comphy", "serdes";
+	dfx-reg = <0x400280 4>;
 	#phy-cells = <2>;
+	sar-data = <&cpm_sar>;
 	status = "disabled";
 };
 
-- 
1.7.9.5

